content/vendor/guides/ha-guide.md

```diff
@@ -0,0 +1,411 @@
+
+To verify that data is not lost, use pgAdmin as described above. Simply run the query again, and should be presented with the same results. You should also have a separate terminal in which to run:
+
+```shell
+watch -d 'kubectl get pods --all-namespaces -o wide'
+```
+
+The above command will display all of the pods in the cluster and on which node they are running on. By using `watch -d` the output will automatically get refreshed so you can watch pod activity as you delete pods and node.
+
+## Deleting a Pod
+
+If you followed along with the steps and sample application, you should see that the Postres pod is running on Node 01. In this test, we are going to delete the Postgres pod. The expected result is that a new pod will be scheduled (possibly on another node) and all the data in the database will be retained and not lost.
+
+To delete the Postgres pod deployed with the sample app, run the following command:
+
+```shell
+kubectl delete pod app-direct-postgresql-0
+```
+This command will schedule a new pod and then terminate the existing one. Verify that the new pod is up and running and check that the data is still there by using pgAdmin. You can also use the various routes in the sample app to verify database connectivity from the app.
+
+## Shutting Down a Node
+
+The next command will test the stability of the cluster when a node is removed. There are a few things you need to keep in mind:
+
+- The Postgres instance in this application, and the application itself are not configured or set up for High Availability. This test is meant to show the resiliancy capabilities of the Kubernetes embedded cluster installed by KURL and configured through Kots.
+
+- As described in the [Kubernetes Documentation](https://kubernetes.io/docs/concepts/architecture/nodes/#condition), there is a default 5 minute timeout duration. This means that there will be 5 minutes between the time the node goes down and that Kubernetes will finally decide that the node is not coming back and will schedule pods on this node to another node.
+
+To simulate a node becoming unresponsive or simply crashed, you can simply stop the VM corresponding to the node. You are likely not to see much change in the pods during the first five minutes, other than some pods erroring out. After 5 minutes or so, you will start to see pods being terminated and scheduled on the remaining nodes. 
+
+In this scenario, the downtime for Postgres was about 6 - 7 minutes. This includes the Kubernetes time out of 5 minutes, 1 minute or less for ekco to detect the node being evicted, and the remainig time is for Postgres to start up on a new node.
+
+
+## Troubleshooting
+
+While the above process has been tested several times and tested succesfully the retention of data, there were a couple of issues that you may run into and are documented below
+
+### Postgres pod will not come up after deleting the pod
```

Comment:
I love the inclusion of a `Troubleshooting` section, more of our guides should have this! I noticed there's a little inconsistency in the cases, this first one uses `I` and `would` to describe a thing that happened, where as the second one is in the second/person with `you` and `may` to describe potential cases. What do you think about consolidating to one style?

Reply:
I updated the entire section, hopefully it is more consistent